src/main/kotlin/com/github/vol0n/utbotcppclion/ui/UTBotProgress.kt

```diff
@@ -0,0 +1,47 @@
+package com.github.vol0n.utbotcppclion.ui
+
+import com.intellij.openapi.progress.TaskInfo
+import com.intellij.openapi.progress.util.AbstractProgressIndicatorExBase
+import com.intellij.openapi.wm.ex.StatusBarEx
+import com.intellij.openapi.wm.ex.WindowManagerEx
+import kotlinx.coroutines.Job
+
+class UTBotRequestProgressIndicator(val name: String, var requestJob: Job? = null) : AbstractProgressIndicatorExBase() {
+    val task = UTBotRequestTaskInfo(name)
+
+    init {
+        isIndeterminate = false
```

Comment:
What does that mean? Could you describe in comments please as the Intellij documentation is not verbose on it. I don't understand what does that mean:
>whether the progress is indeterminate and displays no fractions

Or if you found this code in some example, please provide the link to the example

Reply:
when progressIndicator can indicate the fraction of work remaining it must be set as isIndeterminate = false
<img width="1242" alt="Screenshot 2021-12-06 at 00 38 03" src="https://user-images.githubusercontent.com/64473908/144764829-2ccec4d3-4b2c-4862-bc63-0364bdc5c102.png">